src/airflow/airqo_etl_utils/airqo_utils.py

```diff
@@ -968,5 +968,6 @@ def save_faulty_devices(data: pd.DataFrame):
         client = pm.MongoClient(configuration.MONGO_URI)
         db = client[configuration.MONGO_DATABASE_NAME]
         records = data.to_dict("records")
+        db.faulty_devices.delete_many({})
```

Comment:
@Mnoble-19 , could you shed more light on this? This looks to be an "expensive" operation. On which DB are we storing these records?